userguide/dailyuse/libertine.rst

```diff
@@ -105,78 +115,95 @@ Tips
+You can also make deep links if you only want parts of your SD-card available in the container.
+In this case just add the entire path to the directory you want to bind mount::
 
   libertine-container-manager configure -i CONTAINER-IDENTIFIER -b add -p /media/phablet/ID-OF-SD/directory/you/want
   
-This will not allow the container access to any of the directories earlier in the path for anything other than accessing your mounted directory.
+This will not give the container access to any of the directories earlier in the path for anything other than accessing your mounted directory.
     
-In order to use the SD-card as extra space for your container, make sure first to format it using ext4 or similar.
-There is a mis-feature in udisk2 that mounts SD-cards (showexec) that ensures only files ending in .bat, .exe or .com can be executed from the drive if it is (v)fat formatted. This has been changed in other distributions allowing any file to have execute priviliges, but not ubuntu. The reccomended workaround is to add a udev rule to control how to mount a card with a given id, but since the udev rules are on the read only port on touch, this is not possible.
+In order to use the SD card as extra space for your container, make sure to first format it using ext4 or similar.
+There is a mis-feature in UDisks2 that mounts SD-cards (showexec), ensuring only files ending in .bat, .exe or .com can be executed from the drive if it is (V)FAT formatted.
+This has been changed in other distributions, allowing any file to have execute priviliges, but not in Ubuntu.
+The recommended workaround is to add a udev rule to control how to mount a card with a given ID, but since the udev rules are on the read-only port on Ubuntu Touch, this is not possible.
 
 Shortcuts
 ^^^^^^^^^
 
-If you want, you can add aliases for command line tools. Add lines like the following ones to your ``~/.bash_aliases``::
+If you want, you can add aliases for command-line tools.
+Add lines like the following ones to your ``~/.bash_aliases``::
 
     alias git='libertine-launch -i CONTAINER-IDENTIFIER git'
     alias screenfetch='libertine-launch -i CONTAINER-IDENTIFIER screenfetch'
 
 Background
 ----------
 
-A display server coordinates input and output of an operating system. Most Linux distributions today use the X server. Ubuntu Touch does not use X, but  a new display server called Mir. This means that standard X applications are not directly compatible with Ubuntu Touch. A compatibility layer called XMir resolves this. Libertine relies on XMir to display desktop applications.
-
-Another challenge is that Ubuntu Touch system updates are released as OTA images. A consequence of this is that the root filesystem is read only. Libertine provides a container with a read-write filesystem to allow the installation of regular Linux desktop applications.
+A display server coordinates input and output of an operating system.
+Most Linux|GNU distributions today use the X server.
```

Comment:
sigh. really? are we going down that rabbit hole? oookay, I'm not opposed to giving credit. I guess we can do the gnu linux thing. I just hope it doesn't mess with the readability too much. 

But then let's stick to typical formulation. I am used to see "GNU/Linux" or "GNU Linux", I have never seen "Linux|GNU". Let's stick to "standard" way that will disturb the reading flow as little as possible. 

On a different note - the claim that most distros use X is slowly becoming untrue. Maybe not fully yet, but we could simply remove the sentence. It is irrelevant for the explanation of Libertine whether or not other distros use X or not

Reply:
Linux piped into GNU makes more sense to me. Also the ordering is nice that way. GNU/+ is used by the FSF in some self of self-crediting way. Removing.

Reply:
Mhm. I'm still not convinced. Frankly, whether the *form* of the expression "makes sense" is *to me* a non-issue. The particular *content* doesn't make sense to me. Yes, what people normally call Linux is much more than just the Kernel. But it is also more than the Linux kernel and the GNU system. Yes, Linux + GNU can be called an "Operating System", but, so what. Almost nobody uses a bare OS, without graphics, without network management, without systemd, without a browser, etc, etc. So, the only thing that's left for me here is to say: using the expression GNU/Linux can serve as a symbol for "all those fabulous Free and Open Source Software projects that together give me this well rounded system". And on this symbolic level I really don't see the point in coming up with a new style to express the combination of GNU and Linux, even if for some specific case it might "make more sense". For the symbolic use, I really want to use that same formulation that everybody else uses.

Reply:
What normally happens and what you can conflate it into based on derived assumptions of what it is is not my gripe.
To make the distinction from for example vendor Android Linux, you need some moniker to forward user freedom.
This is important from toybox onwards, even if that has no "GNU" components.
It doesn't undermine Linux to use it this way, because the Linux kernel is neutral.

Precisely by not going with the force-fed convention here, it makes people think.

There are plenty more embedded distributions of "linux" that aren't "Linux|GNU" in this way, and they don't use the Xorg.